ale/drivers/selene_drivers.py

```diff
@@ -694,7 +992,7 @@ def ephemeris_start_time(self):
         : float
           ephemeris start time of the image
         """
-        return spice.sct2e(self.spacecraft_id, self.spacecraft_clock_start_count)
+        return spice.scs2e(self.spacecraft_id, self.spacecraft_clock_start_count)
```

Comment:
A doc string comment here or in spacecraft_clock_start_count that this needs to be a string (and not numeric ticks) might be a good idea?

Reply:
There is actually a difference in some spacecraft. In Kaguya TC it has more to do with the formatting of the `spacecraft_clock_start_count`. When `spacecraft_clock_start_count` is in the "integer" format and you try to wrap it as a string the string routine is unhappy with the value. 

The default assumption is that the `spacecraft_clock_start_count` will be the string formatted version. For KaguyaTC it was the integer format which required the change. Kaguya MI which is where this change is, seems to have the string formatted `spacecraft_clock_start_count`